index.html

```diff
@@ -18,12 +18,23 @@
       href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap"
       rel="stylesheet"
     />
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){window.dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'GA_MEASUREMENT_ID');
+  gtag('config', 'GA_MEASUREMENT_ID', { 'send_page_view': false });
+
+</script>
     <title>Portifolio</title>
```

Comment:
- [ ] To make your page more accessible, your title should adequately and briefly describes the content of the page. Please kindly fix that. 

![youmi](https://user-images.githubusercontent.com/65852621/139925666-e9d2cb88-3b34-4ba6-b90f-42f83df1d222.png)